orb.yaml

```diff
@@ -78,11 +84,15 @@ commands:
           command: |
             git clone https://github.com/navikt/deployment-cli
       - decrypt-private-key
+      - unless:
+          condition: << parameters.enable-vars >>
+          steps:
+            - run: echo "{}" > << parameters.template-vars >>
```

Comment:
Smart way of falling back to empty vars under the silly CircleCI constraints.

What happens if `template-vars` is not set here?

Reply:
Meant to include that in my last commit, now I've added a default value for `template-vars` so if someone sets `enable-vars` to `true` and doesn't provide a filename it will just create an empty file.